vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Supplying usage for ports
+description: Guidance for handling usage files.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 06/30/2023
+---
+# Suppyling usage for ports
+
+## Overview
+
+Providing usage documentation for ports allows users to easily adopt them in their
```

Comment:
Would it be useful to say what a port even is as way of introduction? The article assumes that you know what they are. Maybe they do know, but maybe they don't.

Reply:
"Port" is what we call the set of scripts and metadata that vcpkg uses to install a package/library. At a minimum, a port is a directory containing a `porfile.cmake` script and a `vcpkg.json` metadata file, but it can also contain patch files, other helper scripts and/or the usage file. We assume that users are familiar with the term since we use it extensively in the rest of the documentation, but we don't have an article defining what "port" means (perhaps we should create one?).

Reply:
We should note a task to create a glossary topic that defines things like a port. But since that's not the purpose of this doc, maybe we could shorthand it with something like: "Providing usage documentation for ports (the collection of scripts and metadata vcpkg uses to install a package/library) allows users...